### PR TITLE
Add noteslib

### PR DESCRIPTION
**Pull request by oblute**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by oblute** [comment]
@benhuff @mwilson8 agree to be maintainers?

File: recipes/noteslib/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  sha256: "ca2a4de4a6982785a0160f96085fbc83630fdf0a1f8ed5a9c88a4dd5c3e93ca6"
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  skip: True  # [unix or py2k]
+
+requirements:
+  host:
+    - pip ==18.1
```
Comment (CJ-Wright) [inline recipes/noteslib/meta.yaml]: why the pip pin?

Comment (oblute) [reply recipes/noteslib/meta.yaml]: @CJ-Wright I was getting a backend failure `raise BackendUnavailable pip._vendor.pep517.wrappers.BackendUnavailable` in all commits before `41f6680` ...this thread https://github.com/pypa/pip/issues/6164 showed a work around by reverting back to 18.1. All else showed skipping Windows, but this package is made exclusively for Windows and was released (without further update) when pip's most recent release was 18.1

File: recipes/noteslib/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "ca2a4de4a6982785a0160f96085fbc83630fdf0a1f8ed5a9c88a4dd5c3e93ca6"
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
```
Comment (marcelotrevisani) [inline recipes/noteslib/meta.yaml]:
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv"
```

File: recipes/noteslib/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,42 @@
+{% set name = "noteslib" %}
+{% set version = "2.1.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "ca2a4de4a6982785a0160f96085fbc83630fdf0a1f8ed5a9c88a4dd5c3e93ca6"
```
Comment (marcelotrevisani) [inline recipes/noteslib/meta.yaml]: Could you please remove the quotes from ``sha256``?
```suggestion
  sha256: ca2a4de4a6982785a0160f96085fbc83630fdf0a1f8ed5a9c88a4dd5c3e93ca6
```

**Comment by oblute** [comment]
@marcelotrevisani above changes made

**Comment by oblute** [comment]
@conda-forge/help-python this recipe is ready for final review

**Pull request merged**